libs/cart/src/selectors/cart/cart.selector.spec.ts

```diff
@@ -658,6 +678,34 @@ describe('Cart | Selector | Cart', () => {
     });
   });
 
+  describe('selectItemAdding', () => {
+    describe('when the cart item operations have completed', () => {
```

Comment:
"cart item add operations"

Reply:
fixed

Reply:
oh my bad I don't know why I didn't see this fix initially